kubetest2-gce/deployer/build.go

```diff
@@ -34,14 +34,18 @@ func (d *deployer) Build() error {
 
 	if d.LegacyMode {
 		// this supports the kubernetes/kubernetes build
-		klog.V(2).Info("starting the legacy build")
-
-		cmd := exec.Command("make", "bazel-release")
-		exec.InheritOutput(cmd)
-		cmd.SetDir(d.RepoRoot)
-		err := cmd.Run()
+		klog.V(2).Info("starting the legacy kubernetes/kubernetes build")
+		version, err := d.BuildOptions.Build()
 		if err != nil {
-			return fmt.Errorf("error during make step of build: %s", err)
+			return err
+		}
+		// append the kubetest2 run id
+		version += "+" + d.commonOptions.RunID()
```

Comment:
if version already contains "+", this should append `"+" + d.commonOptions.RunID()`, otherwise we end up with invalid docker tags

Reply:
this is the GCE deployer not GKE, we might want similar changes here too though. yet to test this out for upstream jobs.

Reply:
the double-plus version results in an invalid docker tag regardless of deployer